core/src/main/scala/com/salesforce/op/OpWorkflowModelReader.scala

```diff
@@ -236,15 +236,11 @@ private object WorkflowFileReader {
   def loadFile(pathString: String)(implicit conf: Configuration): String = {
     val path = new Path(pathString)
     val fs = path.getFileSystem(conf)
-    val allFiles = fs.listFiles(path, false)
-    var partPath: Option[Path] = None
-    while (allFiles.hasNext) {
-      val p = allFiles.next().getPath
-      if (p.getName.startsWith("part-00000")) {
-        partPath = Option(p)
-      }
-    }
-    val finalPath = partPath.getOrElse(path)
+
+    val partFile = new Path(pathString, "part-00000")
```

Comment:
better configure the part file name on the write call path so we don't have to guess.

Reply:
it will always be this but the compression codec will determine the extension